runtime/runtime/src/actions.rs

```diff
@@ -883,14 +896,17 @@ pub(crate) fn check_account_existence(
                 .into());
             } else {
                 // TODO: this should be `config.implicit_account_creation`.
-                if config.wasm_config.implicit_account_creation && account_id.is_implicit() {
-                    // If the account doesn't exist and it's 64-length hex account ID, then you
+                if config.wasm_config.implicit_account_creation
+                    // TODO(eth-implicit) Change back to is_implicit() when ETH-implicit accounts are supported.
```

Comment:
In this and several other places, I changed `account_id.get_account_type().is_implicit()` to `account_id.get_account_type() == AccountType::NearImplicitAccount`. This is because in the new version of `near-account-id` we now have that `is_implicit()` returns true for `EthImplicitAccount`, but we want to treat `EthImplicitAccount` as `NamedAccount` in this PR. See [PR discussion](https://github.com/near/near-account-id-rs/pull/14/#discussion_r1382115060).